pom.xml

```diff
@@ -477,26 +476,25 @@
             <artifactId>okhttp</artifactId>
             <version>3.12.12</version>
         </dependency>
-        <!-- TODO many of the following are suspect, should be using core bom -->
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>${slf4j.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>jcl-over-slf4j</artifactId>
-            <version>${slf4j.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>log4j-over-slf4j</artifactId>
-            <version>${slf4j.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
-            <version>${slf4j.version}</version>
+            <scope>provided</scope>
         </dependency>
```

Comment:
these should not be here.  they are defined in the Jenkins-bom which is automatically consumed here.
Also the version seems to be higher than provided by the version of jenkins being used - so likely something may not work correctly (but hopefully it will "be fine" :)

Reply:
this has been moved to `<dependencies>` sections and marked as `provided`